token-lending/js/src/constants.ts

```diff
@@ -1,10 +1,13 @@
 import { PublicKey } from '@solana/web3.js';
 import BigNumber from 'bignumber.js';
 
-export const LENDING_PROGRAM_ID = new PublicKey('6TvznH3B2e3p2mbhufNBpgSrLx6UkgvxtVQvopEZ2kuH');
+export const LENDING_PROGRAM_ID = new PublicKey('AhTXZQVzdtZjbUwMYhti1EggUx778n72kmgP6DT6xURY');
```

Comment:
Sure thing. I only changed it because I made changes to the program (eg changing the account order in init instruction) and without re-deploying the program I couldn't test the cli.

Once we merge the PR with the bug fixes and you update the original program, there shouldn't be a need for this.